templates/partials/nav.html.twig

```diff
@@ -2,4 +2,6 @@
 
 <ul class="navigation">
     {{ macros.nav_loop(pages.children, false, false) }}
+    
+    {{ macros.link_loop(theme_var('show_external_links'), theme_var('external_links')) }}
```

Comment:
This works, but can we change it to use `config.theme.VAR_NAME` just to be consistent with the notation I've used in other templates?

Reply:
Made the change.